receiver/kafkareceiver/zipkin_unmarshaler_test.go

```diff
@@ -89,6 +95,12 @@ func TestUnmarshalZipkin(t *testing.T) {
 			bytes:       thriftTransport.Buffer.Bytes(),
 			expected:    tdThrift,
 		},
+		//{
+		//	unmarshaler: newZipkinJSONv1Unmarshaler(),
+		//	encoding:    "zipkin_v1json",
+		//	bytes:       jsonv1Bytes,
+		//	expected:    td,
+		//},
```

Comment:
I would prefer not having commented out code in the test, is there a reason for having this?